.github/workflows/linting.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - ci/add-python-39-310-311-to-github-workflows
```

Comment:
Any reason this branch would need to remain in the testing path?

Reply:
Oops, that was a mistake. I added my own working branch to test the workflow on my fork...

Just removed this line.